src/main/java/frc/robot/subsystems/drive/Drivetrain.java

```diff
@@ -146,7 +150,7 @@ public ChassisSpeeds getSpeeds() {
 
   /** stops the drivetrain's movement */
   public void stop() {
-    move(new Vector(0, 0), 0);
+    move(Vector.Origin, 0);
```

Comment:
This might have the unintended side-effect of rotating to 0 degrees.

Reply:
The move method takes a direction to move in and an amount to rotate, not a target angle